src/data/path_spawn_data.c

```diff
@@ -349,27 +349,27 @@ BombKartSpawn D_800DCC08[][NUM_BOMB_KARTS_MAX] = {
 };
 
 struct _struct_D_800DD9D0_0x10 D_800DD9D0[] = {
-    { 0x0258, 0x0001, 0x0001, 0x0001, 0x0001, { 0x0000, 0x0000, 0x0000}},
```

Comment:
This struct contains the size of the course path.

So for instance, Yoshi's valley has (iirc) 4 paths. So it should have for sizes for that course. If you measure the size of the course path in course_data.inc.c you'll find them to be the same. I don't remember what the other values do.

However, they are all zero. So presumably they do nothing.